src/cursesport.cpp

```diff
@@ -594,6 +598,19 @@ int clearok(WINDOW *win)
     return 1;
 }
 
+int redrawwin(WINDOW* win)
+{
+    if (! win) {
+        return 0;
+    }
+    win->draw = true;
+    for (int i = 0; i < win->height; i++) {
```

Comment:
Because it doesn't work. 

clearok updates `stdscr` without touching `win` and I don't know if it's intended or not, but calling it doesn't fix black screen issues.

(There's also s problem with clearok that  its proper declaration is `int clearok(WINDOW *win, bool bf);`, not `clearok(WINDOW *win)`)